lib/pluginChangelogParser.js

```diff
@@ -0,0 +1,90 @@
+const fs = require('mz/fs');
+const JSZip = require('jszip');
+const marked = require('marked');
+
+module.exports = class PluginChangelogParser {
+
+    /**
+     * @param {boolean} compileHTML (optional)
+     */
+    constructor(compileHTML) {
+        this.compileHTML = compileHTML === true;
+    }
+
+    /**
+     * @param {String} filePath
+     * @return {Promise}
```

Comment:
Although there is no official JSDoc syntax for async functions yet, I think we should specify the return type based on the source code here, not `Promise`.